# Modeling single cell RNA with optimal control

### Intro
The starting idea considers cells, microglia or neurons, that are healthy or diseased and the question is how to model the transciptomics in order to go from a state to another. This idea will get updated so stay tuned!

The current way to go now is to check some literature and then look into current methods so we can have a couple of baselines to compare with and then we implement the optimal control framework.



### Literature 
- [SERGIO: A Single-Cell Expression Simulator Guided by Gene Regulatory Networks (2020)](https://www.sciencedirect.com/science/article/pii/S2405471220302878) :ballot_box_with_check: :heavy_check_mark: $\checkmark$ 
- [DeepCpG: accurate prediction of single-cell DNA methylation states using deep learning (EBI 2017)](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1189-z)





##### Legend for checkmarks
:ballot_box_with_check: Ionelia checkmark
:heavy_check_mark: Ryan checkmark
$\checkmark$ Manuel checkmark
